Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.h

```diff
@@ -151,6 +151,9 @@ class CLuaVehicleDefs : public CLuaDefs
     static std::variant<bool, CVector>                              OOP_GetVehicleDummyPosition(CClientVehicle* vehicle, eVehicleDummies dummy);
     static bool                                                     ResetVehicleDummyPositions(CClientVehicle* vehicle);
 
+    static std::variant<bool, std::array<std::array<float, 3>, 4>>    GetVehicleEntryPoints(CClientVehicle* vehicle);
+    static std::variant<bool, std::array<CVector, 4>>                 OOP_GetVehicleEntryPoints(CClientVehicle* vehicle);
```

Comment:
Whats the point of OOP function?

Reply:
To return the array of vectors, with the non-OOP version returning an array of floats. The argument parser cannot handle this case, therefore separate functions _are_ necessary.